src/components/DateInput/index.js

```diff
@@ -6,43 +6,43 @@ export class DateInput extends React.Component {
     constructor(props) {
         super(props);
         this.state = {
-            start: "2002-05-06",
-            end: Moment().format('YYYY-MM-DD'),
+            startDate: "2002-05-06",
+            endDate: Moment().format('YYYY-MM-DD'),
         }
     }
 
-    startChange = (e) => {
+    startDateChange = (e) => {
         this.setState({
-            start: e.target.value
+            startDate: e.target.value
         });
     }
-    endChange = (e) => {
+    endDateChange = (e) => {
         this.setState({
-            end: e.target.value
+            endDate: e.target.value
         });
     }
     componentDidMount(){
-        this.props.getLaunches(this.state.start, this.state.end);
+        this.props.getLaunches(this.state.startDate, this.state.endDate);
     }
     render(){
 
         return (
             <form className="dateInput">
                 <label htmlFor="startDate" className="dateLabel">Start Date:</label>
                 <input type="date" id="startDate"
-                    value={this.state.start}
-                    onChange={this.startChange}
-                    max={this.state.end}
+                    value={this.state.startDate}
+                    onChange={this.startDateChange}
+                    max={this.state.endDate}
```

Comment:
I'm curious about `max`. Do you need to specify this?

Reply:
min and max limit the input options to only valid date ranges, so the drop down menu won't list invalid dates.

https://www.w3schools.com/tags/att_input_max.asp